main/introduction.bs

```diff
@@ -12,4 +12,8 @@ Issue: Explain that this specification is not documentation;
 
 ## Definitions ## {#definitions}
 
-Issue: Write Definitions section.
+A <dfn export>data pod</dfn> is a place for storing documents,
```

Comment:
Agreed that such things might be good. Created an issue for tracking this: https://github.com/solid/specification/issues/16

I just wrote two quick definitions because I needed them for cross-referencing purposes. (The "data pod" term will now be linked up.)